src/components/PersonLink.tsx

```diff
@@ -0,0 +1,70 @@
+import cn from 'classnames';
+import { Link, useParams, useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const [searchParams] = useSearchParams();
+
+  const params = useParams();
+
+  const { sex, name, born, died, fatherName, motherName, slug } = person;
+
+  const parents = people.filter(
+    p => p.name === motherName || p.name === fatherName,
+  );
```

Comment:
The `parents` array is being filtered twice - once to get all parents and then individually to get `mother` and `father`. This could be improved for performance by directly assigning `mother` and `father` in one iteration.